apps/src/templates/lessonOverview/LessonStandards.jsx

```diff
@@ -2,6 +2,29 @@ import PropTypes from 'prop-types';
 import React, {PureComponent} from 'react';
 import _ from 'lodash';
 import {standardShape} from './lessonPlanShapes';
+import color from '@cdo/apps/util/color';
+import Radium from 'radium';
+
+export const styles = {
+  frameworkName: {
+    fontFamily: "'Gotham 5r', sans-serif",
+    fontWeight: 'bold',
+    color: color.dark_charcoal
+  },
+  categoryShortcode: {
+    fontFamily: "'Gotham 7r', sans-serif",
+    fontWeight: 'bold',
+    color: color.link_color,
+    ':hover': {
+      textDecoration: 'underline'
+    }
```

Comment:
I recreated link styles here instead of using the `a` element, because the `a` element swallows click events, preventing the summary/details from collapsing/expanding.